src/pages/[redirect]/index.tsx

```diff
@@ -0,0 +1,96 @@
+import fetchOriginalUrl from '@/utils/fetchOriginalUrl';
+import RedirectIcon from '../../../public/assets/icons/redirect';
+import { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import Head from 'next/head';
+import { TINY_SITE } from '@/constants/url';
+
+const Redirect = () => {
+    const router = useRouter();
+    const { redirect: shortUrlCode } = router.query as { redirect: string };
+    const [originalUrl, setOriginalUrl] = useState('');
+    const [timer, setTimer] = useState(5);
+    const [isPremiumUser, setIsPremiumUser] = useState(false);
+    const [showTooltip, setShowTooltip] = useState(false);
+
+    useEffect(() => {
+        setIsPremiumUser(false);
```

Comment:
why are we doing `setIsPremiumUser(false);` everytime in useEffect

Reply:
In the future, we may consider offering a paid plan as an option.

Reply:
agreed but why do we need to set this state value false every time in the useEffect

Reply:
Linter is flagging an unused variable, so I set it to false again.

Reply:
a better approach would have been to use `// eslint-disable-next-line` and comment that it will be used in future.

Reply:
Done